### PR TITLE
Glide up quick

### PR DESCRIPTION
**Pull request by emilevauge**
This PR add `--quick` to `glide up`.
Fixes the issue during `glide up`:
```
[INFO] Setting version for github.com/codegangsta/inject to 33e0aa1cb7c019ccc3fbe049a8262a6403d30504.
[ERROR] Dependency /go/src/github.com/emilevauge/traefik/vendor/github.com/docker/docker/autogen/dockerversion failed to resolve: lstat /go/src/github.com/emilevauge/traefik/vendor/github.com/docker/docker/autogen/dockerversion: no such file or directory.
[ERROR] Failed to retrieve a list of dependencies: lstat /go/src/github.com/emilevauge/traefik/vendor/github.com/docker/docker/autogen/dockerversion: no such file or directory
The command '/bin/sh -c glide up' returned a non-zero code: 1
make: *** [build] Error 1
```

cc @vdemeester @ldez

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -136,7 +136,7 @@ import:
   - package: gopkg.in/fsnotify.v1
     ref:     96c060f6a6b7e0d6f75fddd10efeaca3e5d1bcb0
   - package: github.com/mailgun/manners
-    ref:     37136f736785d7c6aa3b9a27b4b2dd1028ca6d79
```
Comment (vdemeester) [inline glide.yaml]: Is this change related ? :trollface:

**Comment by vdemeester** [comment]
LGTM 🐮 assuming it's green :angel:

File: glide.yaml (hunk 1 above)
Comment (emilevauge) [inline glide.yaml]: Yeah of course!!!
This fixes an error during `glide up` too ;)
